chainerx_build_helper.py

```diff
@@ -7,11 +7,25 @@
 import re
 import subprocess
 import sys
+import warnings
 
 import setuptools
 from setuptools.command import build_ext
 
 
+def get_env_boolean(var: str, default: str, env=None):
```

Comment:
The function is never called with `env != None` but could you explain why it's needed?

Reply:
base on personal experience, sometimes one might needs to use a copied env dict

Reply:
I see, thanks for the explanation. It sounds like slight overengineering though and I'd prefer keeping the interface simple as it's easy to extend it afterwards.